recipes/huffman/all/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+﻿cmake_minimum_required (VERSION 3.4)
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9" CACHE STRING "Minimum OS X deployment version")
+project (huffman)
```

Comment:
not super important, but a CMakeLists shoud always start with `cmake_minimum_required` then `project`, and nothing between these 2 commands.